src/Commands/Import/SiteCommand.php

```diff
@@ -16,18 +16,18 @@ class SiteCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Import a site archive onto a Pantheon site
+     *  Imports a site archive (code, database and files) to the site.
```

Comment:
Does Pantheon have a stance on the Oxford comma, because I've included it everywhere possible in Terminus.